### PR TITLE
FIX attribute field name

### PR DESCRIPTION
**Pull request by bealdav**
Typo error in v8:

s/sting/string

Fixed in v9

Thanks for your nice module
ping @jbeficent @JosDeGraeve @carlosdauden

### CODE REVIEW

File: stock_putaway_product/models/product_putaway.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def get_default_inventory_id(self):
 
     putaway_id = fields.Many2one(
         comodel_name='product.putaway',
-        sting='Put Away Method',
+        string='Put Away Method',
```
Comment (tafaRU) [inline stock_putaway_product/models/product_putaway.py:62]: However, `sting` sounds better! :rofl: 
But joking apart, it's strange no error has been raised during CI builds.

**Review by tafaRU — APPROVED** [review]

File: stock_putaway_product/models/product_putaway.py (hunk 1 above)
Comment (bealdav) [reply stock_putaway_product/models/product_putaway.py:62]: yes but 'string' looks nice too: http://www.dessus-dessous.fr

ping @sebastienbeau @bguillot

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Comment by bealdav** [comment]
ping PSC @Cedric-Pigeon @lasley 

could you merge this trivial typo PR ? thanks a lot

**Comment by bealdav** [comment]
thanks a lot @Cedric-Pigeon

**Pull request merged**